ros2/naive/src/node.cpp

```diff
@@ -0,0 +1,19 @@
+#include "naive/incrementer.hpp"
+
+#include "rclcpp/rclcpp.hpp"
+
+int main(int argc, char **argv) {
+
+	rclcpp::init(argc,argv);
+
+	auto const node = std::make_shared<rclcpp::Node>("incrementer");
```

Comment:
East const is inconsistent with previous uses of `const`. Let's pick one style and stick with it. Looks like all other uses of const use const west.

Reply:
I should convert this repo to east const because it's the correct const, but I can do that all at once in a later PR. Consistency is best

Reply:
> it's the correct const

> Consistency is best

Don't forget that consistency across PickNik repos is valuable as well. Let's not make any huge deviations from existing PickNik style without good reason.

Reply:
o. lab0 uses east const...

Reply:
Update: Just learned that clang-format-14 lets you enforce one style versus the other so as long as we're using clang-format-14 to enforce a given style, I do not care what we pick.

Reply:
I didn't know there was such a heated debate between west const and east const. This reference helped clear up some stuff for me: https://mariusbancila.ro/blog/2018/11/23/join-the-east-const-revolution/

Regardles, I've switched the style to east const for the PR.